File: java/src/MyBot.java
import utils.Pathfinder;

import java.util.ArrayList;
import java.util.HashMap;

public class MyBot extends Bot{

    public String get_name(){
        // Find a name for your bot
        return "Bot name";
    }

    public HashMap<String, String> turn(String game_state, HashMap<String, Object> character_state, ArrayList<HashMap<String, Object>> other_bots){
        // Your bot logic goes here.
        return command.idle();
    }
}

File: src/bot/MyBot.py
from src.bot.Bot import Bot


class MyBot(Bot):

    def __init__(self):
        super().__init__()

    def get_name(self):
        # Find a name for your bot
        return 'My bot'

    def turn(self, game_state, character_state, other_bots):
        # Your bot logic goes here
        super().turn(game_state, character_state, other_bots)
        return self.commands.idle()
